### PR TITLE
v3(docs) Document migration path from space summary to using fields.

### PR DESCRIPTION
**Pull request by FelisiaM**
[#176251091](https://www.pivotaltracker.com/story/show/176251091)

### CODE REVIEW

File: docs/v3/source/includes/upgrade_guide/conceptual_changes/_resource_summaries.md [hunk 1]
```diff
@@ -21,10 +21,42 @@ filter by the parent resource. See below for examples of summaries in V3.
...
+- To fetch all service offerings in a space use `GET
+  /v3/service_offerings?space_guids=<space-guid>`. 
 - To fetch all service instances in a space use `GET
   /v3/service_instances?space_guids=<space-guid>`. Use the
   experimental `fields` parameter to include related information in the response
   body.
+  
+##### Replacing the space summary response for service instances
+
+Similar fields to what `/v2/spaces/:guid/summary` was offering for services are available in using endpoint in v3.
```
Comment (pivotal-marcela-campo) [inline docs/v3/source/includes/upgrade_guide/conceptual_changes/_resource_summaries.md:33]: "for services are available in using endpoint in v3." should be something like "for services are available using the `service_instances` in `v3`."

File: docs/v3/source/includes/upgrade_guide/conceptual_changes/_resource_summaries.md [hunk 2]
```diff
@@ -21,10 +21,42 @@ filter by the parent resource. See below for examples of summaries in V3.
...
+
+The table below describes the query parameters needed to retrieve some of those fields using `/v3/service_instances` endpoint. 
+Same queries are available on request for single resource.
```
Comment (pivotal-marcela-campo) [inline docs/v3/source/includes/upgrade_guide/conceptual_changes/_resource_summaries.md:36]: Same ***query parameters*** are available on ***the*** request for ***a*** single resource.

File: docs/v3/source/includes/upgrade_guide/conceptual_changes/_resource_summaries.md [hunk 3]
```diff
@@ -21,10 +21,42 @@ filter by the parent resource. See below for examples of summaries in V3.
 
 - To fetch all apps in a space, use `GET /v3/apps?space_guids=<space-guid>`.
   Passing `include=space` will include the space resource in the response body.
+- To fetch all service offerings in a space use `GET
+  /v3/service_offerings?space_guids=<space-guid>`. 
 - To fetch all service instances in a space use `GET
   /v3/service_instances?space_guids=<space-guid>`. Use the
```
Comment (pivotal-marcela-campo) [inline docs/v3/source/includes/upgrade_guide/conceptual_changes/_resource_summaries.md:27]: The `fields` parameter is available for service offerings as well. I wonder if we should repeat that sentence in both or point it out after this list.

**Review by pivotal-marcela-campo — CHANGES_REQUESTED** [review]

File: docs/v3/source/includes/upgrade_guide/conceptual_changes/_resource_summaries.md (hunk 1 above)
Comment (FelisiaM) [reply docs/v3/source/includes/upgrade_guide/conceptual_changes/_resource_summaries.md:33]: actually they are available from different endpoints. I will update it to `are available from endpoints in v3`

**Review by pivotal-marcela-campo — APPROVED** [review]

**Pull request merged**